hub/util/exceptions.py

```diff
@@ -65,6 +65,80 @@ def __init__(self):
         super().__init__("Ensure that len(size_list) + 1 == len(provider_list)")
 
 
+class LoginException:
+    def __init__(
+        self,
+        message="Error while logging in, invalid auth token. Please try logging in again.",
```

Comment:
If Auth token is invalid we should just say that. : Login Error. Invalid Auth Token.

Please remove "Please try logging in again"

Reply:
I disagree. We should give users an actionable error message. Otherwise, they can't do anything to solve this error.

Reply:
Not true right? Logging in with the same token won't help. A better action item is to check the auth token.
This might change with Platform generated API Keys.

Reply:
No, login exception only happens when user tries Hub login and it fails to give a proper token. Logging in again might fix the issue

Reply:
okay makes sense.